packages/apps/plugins/plugin-excalidraw/src/SketchPlugin.tsx

```diff
@@ -115,6 +115,11 @@ export const SketchPlugin = (): PluginDefinition<SketchPluginProvides> => {
       },
       surface: {
         component: ({ data, role }) => {
+          // TODO(burdon): Fix.
+          if (data.object instanceof DiagramType && data.object.canvas?.schema !== EXCALIDRAW_SCHEMA) {
```

Comment:
why not include this condition on the instance of checks below? currently this will break the settings panel because that surface doesn't pass `object`